apps/blockchain/lib/blockchain/block/holistic_validity.ex

```diff
@@ -82,7 +82,6 @@ defmodule Blockchain.Block.HolisticValidity do
       |> check_transactions_root_validity(child_block, block)
       |> check_gas_used(child_block, block)
 
-    # TODO: Check receipts after resolving https://github.com/poanetwork/mana/issues/66
     # |> check_receipts_root_validity(child_block, block)
```

Comment:
Do we have an issue for enabling `check_receipts_root_validity` here or will that be included in [530: store Receipts in the storage](https://github.com/poanetwork/mana/issues/450)

Reply:
I created an issue https://github.com/poanetwork/mana/issues/459